### ISSUE TITLE
Install is broken

### ISSUE THREAD

**Issue by ghost** [sloria/local-repl#11]
No matter, globally or locally trying to install this package results in

```
[1Anpm ERR! Linux 4.10.0-37-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "local-repl"
npm ERR! node v4.7.2
npm ERR! npm  v3.5.2
npm ERR! path /usr/local/lib/node_modules/local-repl/cli.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/local-repl/cli.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/local-repl/cli.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /home/wvxvw/dev/git/mic/frontend/npm-debug.log
```

Upon inspection `/usr/local/lib/node_modules/` doesn't exist (and why should it?), but if trying to install locally, this tries to find:

```
[1Anpm ERR! Linux 4.10.0-37-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "local-repl" "--save-dev"
npm ERR! node v4.7.2
npm ERR! npm  v3.5.2
npm ERR! path /home/wvxvw/dev/git/mic/frontend/node_modules/local-repl/cli.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/home/wvxvw/dev/git/mic/frontend/node_modules/local-repl/cli.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/wvxvw/dev/git/mic/frontend/node_modules/local-repl/cli.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /home/wvxvw/dev/git/mic/frontend/npm-debug.log
```

While `/home/wvxvw/dev/git/mic/frontend/node_modules/local-repl/` also doesn't exist.

NPM version is 3.5.2

**Comment by sloria** [comment]
This is likely an issue with your node dev environment rather than this package. Are you able to install other packages locally? Also, have you tried upgrading npm?

**Comment by sloria** [comment]
@wvxvw Actually, you're right, the release is broken. I'll fix this after work today. Sorry about that.

**Comment by sloria** [comment]
This is fixed now in 3.0.1
